lib/suppressions.h

```diff
@@ -22,6 +22,7 @@
 
 #include "config.h"
 #include "errortypes.h"
+#include "tokenize.h"
```

Comment:
it does not seem this include is necessary here. I would prefer a forward declaration in the header:

    class Tokenizer;

Reply:
Yes, and then include in the .cpp file? Or should we just only take a pointer to the first token? I just looked at some other functions that used the tokenizer and then took the token list from there.

Reply:
hmm.. I don't have a super strong opinion if you should pass `Tokenizer` or `Token`. Both will work fine. Well.. if you pass tokenizer it's more apparent that the whole token list is used. so feel free to keep the interface as is.